test/wsdl-parse-test.js

```diff
@@ -21,4 +27,69 @@ describe(__filename, function () {
       done( err );
     });
   });
+
+  it('should parse multiple xsd files with the same namespace using lodash merge', function (done) {
+    wsdl.set_merge_method('lodash.default');
```

Comment:
It's test code that needs to be removed after being reviewed.  I'm only trying to demonstrate that merging the definitions from two XSDs does not work with lodash merge or lodash defaults.